x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/security/authz/60_resolve_index.yml

```diff
@@ -122,13 +122,6 @@ teardown:
   - match: { data_streams.0.backing_indices.0: $idx0name }
   - match: { data_streams.0.timestamp_field: "@timestamp" }
 
-  # should fail because no_authz_user is not authorized for view_index_metadata
-  - do:
-      catch: forbidden
-      headers: { Authorization: "Basic bm9fYXV0aHpfdXNlcjp4LXBhY2stdGVzdC1wYXNzd29yZA==" } # no_authz_user
-      indices.resolve_index:
-        name: '*'
-
```

Comment:
I'd prefer we change the `data_stream_role2` role to use `monitor` instead of `read` and then kept this test.

Reply:
Sounds good, I pushed a change that does this. I just can't figure out why it fails on the v7 compatibility yet.